### ISSUE TITLE
New connection icon ambigious

### ISSUE THREAD

**Issue by peterschen** [icanos/SMAStudio#37]
The icon for a new connection can mean anything. I actually looked for it for about 10 minutes before figuring that one out. Maybe we can use a more fitting icon here.

**Comment by icanos** [comment]
Yes, definitely. I've been thinking of changing the icon as well as removing one of the buttons since the button is represented in two different locations.

The 'Add connection' button is present in the main toolbar of the application, as well as the toolbar of the Environment Explorer (if that is open). Where do you see the best fit for an icon like that? The connections are listed in the Environment Explorer and due to that, is a part of the EE. But if the EE isn't visible there is no way of adding a new connection and quite hard to find the new connection button.

Maybe changing it to add a new menu item to "File" in the dropdown menu and keeping the button in the Environment Explorer, or should we keep it as is?
